rdr_service/workflow_management/nph/sms_workflows.py

```diff
@@ -146,6 +147,23 @@ def job_ingestion(self):
         Main method for ingestion jobs.
         """
 
+        # Ensure file is CSV
+        if not str(self.file_path).endswith(".csv"):
+            file_end = self.file_path.split(".")[-1]
```

Comment:
nit: you could probably move `file_end` up a couple lines and then test `if file_end != 'csv'`